java-shared-dependencies/dependency-analyzer/src/main/java/com/google/cloud/exception/DependencyRiskException.java

```diff
@@ -0,0 +1,7 @@
+package com.google.cloud.exception;
+
+public class DependencyRiskException extends Exception{
```

Comment:
I now feel that the retrieving risk is norm for this tool, rather than exception. Let's keep exceptions for network error or invalid inputs.

Reply:
Removed this exception.